test/uk/gov/hmrc/tai/service/TaxCodeChangeServiceSpec.scala

```diff
@@ -36,16 +40,20 @@ class TaxCodeChangeServiceSpec extends PlaySpec with MockitoSugar{
       val sut = createSut
       val nino = generateNino
 
-      val taxCodeHistory = TaxCodeHistory(nino.nino, Seq(TaxCodeRecord("1185L","Employer 1","operated","2017-06-23")))
+      val taxCodeHistory = TaxCodeHistory(taxCodeRecord1, taxCodeRecord2)
 
       when(sut.taxCodeChangeConnector.taxCodeHistory(any())(any())).thenReturn(Future.successful(TaiSuccessResponseWithPayload(taxCodeHistory)))
 
       val result = sut.taxCodeHistory(nino)
-      Await.result(result, 5 seconds) mustBe TaiSuccessResponseWithPayload(taxCodeHistory)
+      Await.result(result, 5 seconds) mustBe taxCodeHistory
```

Comment:
Can we change the `5 seconds` to `5.seconds`? These keep throwing compiler warnings